### PR TITLE
Add provider info and description to the clusterserviceversion

### PR DESCRIPTION
**Pull request by raghavendra-talur**
This ensures that the provider info is not lost on every bundle creation.

### CODE REVIEW

**Comment by raghavendra-talur** [comment]
Merge after #25

**Comment by ShyamsundarR** [comment]
Looks ok, will take a look post 25 is merged.

**Comment by nirs** [comment]
@raghavendra-talur you can add `Fixes #27` to the first commit.

File: config/manifests/bases/recipe.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Recipe is the Schema for the recipes API
+      displayName: Recipe
+      kind: Recipe
+      name: recipes.ramendr.openshift.io
+      version: v1alpha1
+  description: Recipe description. TODO.
```
Comment (nirs) [inline config/manifests/bases/recipe.clusterserviceversion.yaml]: We can use: "Recipe describes a workflow used for a capturing or recovering Kubernetes resources."

(based on https://github.com/RamenDR/ramen/blob/main/docs/recipe.md)

**Review by nirs — APPROVED** [review]

**Pull request merged**